services/submission/src/main/resources/application.yaml

```diff
@@ -47,8 +47,8 @@ services:
         key-password: ${SSL_SUBMISSION_KEYSTORE_PASSWORD}
         key-store: ${SSL_SUBMISSION_KEYSTORE_PATH}
         key-store-password: ${SSL_SUBMISSION_KEYSTORE_PASSWORD}
-        trust-store: ${SSL_VERIFICATION_TRUSTSTORE_PATH}
```

Comment:
I think this should not be changed in submission service. The verification truststore path should be different from federation.

Reply:
Oh, thanks for the catch!